systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -645,4 +648,68 @@ public static void verifyKafkaKraftMetadataLog(final TestStorage testStorage, fi
     private static String buildDirectoryPath(int volumeId, int kafkaIndex) {
         return String.format("/var/lib/kafka/data-%d/kafka-log%d/__cluster_metadata-0", volumeId, kafkaIndex);
     }
+
+    /**
+     * Waits until the Kafka cluster in the given namespace the specified AutoRebalance state is set.
+     *
+     * @param namespaceName         The name of the Kubernetes namespace where the Kafka cluster resides.
+     * @param clusterName           The name of the Kafka cluster.
+     * @param expectedState         The expected {@link KafkaAutoRebalanceState} that is set.
+     */
+    public static void waitUntilKafkaHasAutoRebalanceState(final String namespaceName,
+                                                           final String clusterName,
+                                                           final KafkaAutoRebalanceState expectedState) {
+        TestUtils.waitFor(
+            String.format("waiting for Kafka cluster '%s' in namespace '%s' and KafkaRebalance state: '%s'", clusterName, namespaceName, expectedState),
+            TestConstants.GLOBAL_POLL_INTERVAL,
+            TestConstants.GLOBAL_TIMEOUT,
+            () -> {
+                Kafka kafka = KafkaResource.kafkaClient().inNamespace(namespaceName).withName(clusterName).get();
+                if (kafka == null || kafka.getStatus() == null) {
+                    return false; // Kafka cluster or its status is not available
+                }
+
+                final KafkaAutoRebalanceState currentState = kafka.getStatus().getAutoRebalance() != null
+                    ? kafka.getStatus().getAutoRebalance().getState() : null;
+
+                return currentState != null && currentState.equals(expectedState);
+            }
+        );
+    }
+
+    /**
+     * Waits until the Kafka cluster in the given namespace reaches the specified AutoRebalance mode and brokers.
+     *
+     * @param namespaceName                     The name of the Kubernetes namespace where the Kafka cluster resides.
+     * @param clusterName                       The name of the Kafka cluster.
+     * @param expectedKafkaAutoRebalanceMode    The expected {@link KafkaRebalanceMode} that the Kafka cluster should reach.
+     * @param scalingBrokers                    A list of expected broker IDs that the Kafka cluster should have in AutoRebalance state.
+     */
+    public static void waitUntilKafkaHasExpectedAutoRebalanceModeAndBrokers(final String namespaceName,
+                                                                            final String clusterName,
+                                                                            final KafkaRebalanceMode expectedKafkaAutoRebalanceMode,
+                                                                            final List<Integer> scalingBrokers) {
+        TestUtils.waitFor(
+            String.format("Waiting for Kafka cluster '%s' in namespace '%s' and KafkaRebalance mode '%s' with brokers %s",
```

Comment:
Same as above.
Also, there was a PR some time ago that "standardized" how the log messages are written, so something like:
`"for Kafka cluster {}/{} ...", namespaceName, clusterName`
and we should keep it consistent.

Reply:
Done, thanks.